tracer/src/Datadog.Trace/AppSec/Rcm/IAsmConfigUpdater.cs

```diff
@@ -0,0 +1,17 @@
+﻿// <copyright file="IAsmConfigUpdater.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+using System.Collections.Generic;
+using Datadog.Trace.RemoteConfigurationManagement;
+
+namespace Datadog.Trace.AppSec.Rcm;
+
+internal interface IAsmConfigUpdater
```

Comment:
If we go for a configUpdater, we could drop this interface and all its implementations